### ISSUE TITLE
worker.stopped is False after shutdown?

### ISSUE THREAD

**Issue by urda** [nvie/rq#525]
Given:

```python
with Connection(conn):
    rq_worker = Worker(list(map(Queue, listen)))
    rq_worker.work()

state = rq_worker.stopped
```

Why is `state` set to `False` even after the worker obtains `SIGTERM`? I feel that after leaving the `.work()` line, that should be set to `True` in order for a parent script to know the child was stopped.

**Comment by selwin** [comment]
`stopped` is actually only internally used to indicate that the user has requested the worker to stop. If you want to query the status of the worker, you should use `worker.state` which has these options:

1. started
2. suspended
3. busy
4. idle

I'm open to adding `stopped` state that's set after worker finishes `work()`.

**Comment by nvie** [comment]
Let's keep having just `state` and perhaps rename `stopped` to `stop_requested` for the internal usage?

**Comment by selwin** [comment]
Perhaps `_stop_requested` to emphasize that it's an internal variable.

**Comment by nvie** [comment]
Thanks, fixed in 0de225ec62fffc1f008fdee10e32afe45f7067a9.
